org.uqbar.project.wollok.typeSystem/src/org/uqbar/project/wollok/typesystem/GenericType.xtend

```diff
@@ -0,0 +1,18 @@
+package org.uqbar.project.wollok.typesystem
+
+import org.eclipse.xtend.lib.annotations.Accessors
+import org.uqbar.project.wollok.wollokDsl.WClass
+
+/**
+ * @author npasserini
+ */
+	@Accessors
+class GenericType extends ClassBasedWollokType {
```

Comment:
New "GenericType", representing "raw" generic types, such as List<E>. 
We might end up removing this, as we joined GenericTypeInfo with SimpleTypeInfo, but right now it helps to see where the generic types need special attention.